tools/make/ffi.toml

```diff
@@ -126,30 +120,22 @@ category = "ICU4X Development"
 script_runner = "@duckscript"
 script = '''
 exit_on_error true
-cd ffi/diplomat/cpp/examples/pluralrules
-rm -f ../../../../../target/debug/libicu_capi_staticlib_test.a
-exec --fail-on-error make
-cd ../properties
-rm ../../../../../target/debug/libicu_capi_staticlib_test.a
-exec --fail-on-error make
-cd ../bidi
-rm ../../../../../target/debug/libicu_capi_staticlib_test.a
+cd ffi/diplomat/cpp/examples/properties
```

Comment:
Question: why did you change the order?

Reply:
Grouped examples with the same cargo features to avoid rebuilding.